March Mapping Madness - New space ruin

Astrogem2 opened the pull request:
I have made a new space ruin which is based off of the game Prey, cause mimics are cool, and I think some variety in space ruins would be swell
## About The Pull Request
This pull request is about a new ruin I have made, which I have taken inspiration on by the hit 2017 game Prey, which I enjoy.

![prey 2, the peryer](https://user-images.githubusercontent.com/101240878/224872167-e39ab784-9bf2-4d0a-bf3f-9e277ae1b7e8.png)


### Mapping March
Ckey to receive rewards: Astrogem2

## Why It's Good For The Game
We don't have that many Ruins out in space, and some new ones would be nice to see.

## Changelog
:cl: Astrogem2

add: New space ruin that I have made, which adds some variety to space searching.

/:cl:

Astrogem2 committed:
March Mapping Madness - New space ruin

I have made a new space ruin which is based off of the game Prey, cause mimics are cool, and I think some variety in space ruins would be swell

Astrogem2 committed:
Merge branch 'master' into Prey

Astrogem2 committed:
Update Prey-Ruin.dmm

VladinXXV commented:
Can you include pictures of the ruin in the PR body please? You might also want to expand your changelog entry.

Astrogem2 commented:
ight

Astrogem2 committed:
Changes a small oopsie in a variable I made

Cause I put true instead of 1

san7890 commented:
that asteroid is way too big for so much regular rock, trim down the overall map size or just splinter that big asteroid into a bunch of smaller ones

Astrogem2 commented:
got it

Astrogem2 committed:
New look.

Made it smaller from san's request.

Astrogem2 committed:
Merge branch 'Prey' of https://github.com/Astrogem2/Prey-Ruin into Prey

san7890 commented:
be sure to edit your PR body with the latest version of the map to ensure that newer people who come in and see this PR don't give feedback on an outdated version of the map, thanks

wesoda25 commented:
Love the little simple ones like this 👍

Addust commented:
plating under the windows
hivelords, holy shit rare

Addust commented:
also it needs a ruin config thingymajig (check other ruin PRs!)

Valorium-RB commented:
The foam force crossbow should be renamed to the Huntress Boltcaster, to complete the reference.

Astrogem2 commented:
damn i’m gaming right now, i’ll do these after

Astrogem2 committed:
Put it where it goes, and adds it to ruin config.

I have moved the map to the ruin folder where the other ones are, along with adding it to the ruins config, I also removed an extra gap in the config because it didn't look important. (also the CRLF line endings are being turned into LF endings, I hope that doesn't do anything bad)

Astrogem2 commented:
ok i've done that now i'm pretty sure.

san7890 commented:
they aren't in your PR. push your commit to remote, or commit your staged changes and then push to remote.

Astrogem2 commented:
uhhhhh, ill try to do that i guess.

Astrogem2 commented:
wait i think i made another pr for that by mistake

Astrogem2 committed:
Adds my ruin dmm to the blacklist

After a little while, i now know where this file is, and now its in this pr commit, so it doesn't need another pr to do this :)

Astrogem2 commented:
ok this time its sure to work im pretty sure, i hope

Astrogem2 commented:
oh christ

Astrogem2 committed:
adds little ending space

Linters said to add the space, hope this works.

Astrogem2 commented:
this might fix the one linter error.

Astrogem2 committed:
bug fix in space.dm file

turns out i forgot the "" in the var thing.

Astrogem2 committed:
Merge branch 'Prey' of https://github.com/Astrogem2/Prey-Ruin into Prey

Astrogem2 commented:
ok im praying that this works now.

san7890 commented:
fix your conflicts, and also add a `/obj/effect/baseturf_helper/asteroid/airless` into the ruin's area (maybe making a new area for it as well, it's not that hard) such that if anyone decides to use a grenade, it'll only scrape away the walls of the ruin, and not punch a hole into space.

Astrogem2 committed:
Merge branch 'master' into Prey

Astrogem2 committed:
adds baseturf helper

adds the helper that makes it so that when something explodes, it only removes the walls and now the floors.

Astrogem2 committed:
Merge branch 'Prey' of https://github.com/Astrogem2/Prey-Ruin into Prey

Astrogem2 committed:
fixing linter error

fixes linter error caused by spaces and not tab indent (i hope)

Astrogem2 committed:
Merge branch 'master' into Prey

Maurukas commented on code/datums/ruins/space.dm:
```diff
@@ -356,3 +355,9 @@
+
+/datum/map_template/ruin/space/prey_pod
+	id = "prey"
+	suffix = "prey_pod"
```
```suggestion
	suffix = "prey_pod.dmm"
```

This needs to be the filename

Maurukas reviewed (changes requested):
Looks good. Please fix the file name.

If you'd like, you can give the terminal custom text. Look at `code/modules/mapfluff/ruins/spaceruin_code/dangerous_research.dm` and `TheDerelict.dm` for examples.

Astrogem2 committed:
Update code/datums/ruins/space.dm

Co-authored-by: Vire <66576896+Maurukas@users.noreply.github.com>

Maurukas commented on config/spaceruinblacklist.txt:
```diff
@@ -54,5 +54,6 @@
 #_maps/RandomRuins/SpaceRuins/space_billboard.dmm
 #_maps/RandomRuins/SpaceRuins/spinwardsmoothies.dmm
-#_maps/RandomRuins/SpaceRuins/dangerous_research.dmm
+#_maps/RandomRuins/SpaceRuins/dangerous_research.dm
```
```suggestion
#_maps/RandomRuins/SpaceRuins/dangerous_research.dmm
```

Maurukas committed:
Update config/spaceruinblacklist.txt
